package.json

```diff
@@ -8,7 +8,7 @@
     "chrome-debug": "./lighthouse-core/scripts/manual-chrome-launcher.js"
   },
   "engines": {
-    "node": ">=10.13"
+    "node": ">=12.13.0"
```

Comment:
Should this be `12.0`? Or should CI install `12.13.0`?

Reply:
I always get annoyed when packages pick an unnecessarily specific version, anything wrong with `12.x`?

Reply:
12.13 was the first LTS version, so it seems ok to rely on that as a floor. OTOH I don't see much that was added between 12 and 12.13. `Promise.allSettled`, but anything else?

Reply:
> Promise.allSettled, but anything else

[looks like yes](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#12.13.0), there were a few more things we'll be interested in, like the `Intl.NumberFormat` improvements (like `'unit'`) and the V8 [7.6](https://v8.dev/blog/v8-release-76) and [7.7](https://v8.dev/blog/v8-release-77) releases. Doesn't seem too onerous to use the October 2019 release as a minimum?

Reply:
alright